spec/requests/api/service_requests_spec.rb

```diff
@@ -79,4 +79,120 @@ def expect_result_to_have_user_email(email)
       expect_result_to_have_user_email(@user.email)
     end
   end
+
+  context "authorization" do
+    it "is forbidden for a user without appropriate role" do
+      api_basic_authorize
```

Comment:
This method takes an optional argument with a list of identifiers. So if your resource is not protected, you'll be authorized, but if it is you'll need to pass in the specific identifier you need to authorize you for that resource. :tophat: :rabbit2: :sparkles: , I know